forex-mtl/src/main/scala/forex/rates/errors.scala

```diff
@@ -0,0 +1,8 @@
+package forex.rates
+
+object errors {
+
+  sealed trait RateError extends Exception
```

Comment:
We used to organize this differently like : 
```
sealed trait RateError extends Exception
object RateError {
   final case class RemoteClient(msg: String) extends RateError
}
```
any reason why you change this ?

Reply:
No reason, can do that :+1: